#this is a small test for changing goal_position

int32 goal_position

---

bool result
